bn_mp_to_ubin.c

```diff
@@ -28,6 +41,9 @@ mp_err mp_to_unsigned_bin(const mp_int *a, unsigned char *b)
    s_mp_reverse(b, x);
```

Comment:
~and then also change the API of `s_mp_reverse()` to use `size_t`~ that's already done in #348